cfpb/cfgov-refresh
# Clean up changes left over from CF v4

Short description explaining the high-level reason for the pull request

## Removals

- Removed files that are no longer used

## Changes

- Updated includes to use latest CF classes and markup examples

## Testing

1. run `gulp build`.
2. run `gulp test:acceptance`.
3. navigate throughout the site and check for errors.

## Screenshots

![screen shot 2017-10-13 at 3 08 23 pm](https://user-images.githubusercontent.com/1280430/31564426-60fbbf22-b028-11e7-9b7f-696cfbeb8173.png)

![screen shot 2017-10-13 at 3 09 49 pm](https://user-images.githubusercontent.com/1280430/31564487-97e423ee-b028-11e7-8936-fb7b8aae08c6.png)

![screen shot 2017-10-13 at 3 10 43 pm](https://user-images.githubusercontent.com/1280430/31564515-b487528c-b028-11e7-9e7c-132618712e79.png)


## Notes

- This is only one small part of this cleanup. Breaking it up into multiple PRs to keep them from being huge.

## Checklist
